src/content/docs/en/guides/middleware.mdx

```diff
@@ -154,16 +154,14 @@ A function that intercepts (reads and modifies) the `Response` of a `Request` or
 
 This is an optional argument of `onRequest()`, and may provide the required `Response` returned by the middleware.
 
-
 ### `locals`
 An object containing data from a response that can be manipulated inside middleware. 
 This `locals` object is forwarded across the request handling process and is available as a property to `APIContext` and `AstroGlobal`. This allows data to be shared between middlewares, API routes, and .astro pages. This is useful for storing request-specific data, such as user data, across the rendering step.
 
 `locals` is an object that lives and dies within a single Astro route; when your route page is rendered, `locals` won't exist anymore and a new one will be created. Information that needs to persist across multiple page requests must be stored elsewhere.
 
-In `dev` mode, Astro perform checks and will throw an error if the following conditions are not met:
-- `locals` needs to be serializable: The information must be serializable because storing information that evaluates at runtime is unsafe. If, for example, we were able to store a JavaScript function, an attacker could exploit the victim's website and execute some unsafe code.
-- `locals` can't be overridden: The value of locals needs to be an object, and it can't be overridden at runtime. Doing so would risk wiping out all the information stored by the user.
+In `dev` mode, Astro performs checks and will throw an error if `locals` are overridden.
+The value of locals needs to be an object, and it can't be overridden at runtime. Doing so would risk wiping out all the information stored by the user.
```

Comment:
```suggestion
:::note
The value of `locals` cannot be overridden at run time. Doing so would risk wiping out all the information stored by the user.  In `dev` mode, Astro performs checks and will throw an error if `locals` are overridden.
:::
```
What do we think about this, and making it a note? I feel like swapping the order makes more sense now that there are not two different things that throw an error. Rather, this section is now just about not overriding `locals`, and we back it up by saying there's gonna be an error if you do?

Reply:
Thank you Sarah for the great suggestion! ❤️